tests/unit/helpers_test.sh

```diff
@@ -61,6 +69,10 @@ function test_no_function_is_executed_with_execute_function_if_exists() {
 function test_unsuccessful_unsetIfExists() {
   assertGeneralError "$(Helper::unsetIfExists "fake_function")"
 }
+function test_unsuccessful_unsetIfExists() {
+  # TODO: this is duplicated and it should failed; should be fixed on 'feat/stop-exec-when-found-duplicate'
```

Comment:
?

Reply:
This was an intentional duplicated test to force the error of duplicated test functions. I forgot to remove it, thanks